.github/workflows/check-outdated-terms.yaml

```diff
@@ -39,17 +39,8 @@ jobs:
         OUTPUT_DIR="./outdated"
         
         # Set L10n directory and code
-        case "${L10N_BRANCH}" in
-          dev-ko)          
-          L10N_DIR="./content/ko/"
-          L10N_CODE="ko"
-          ;;
-          
-          #dev-pt)
-          #L10N_DIR="./content/pt-br/"
-          #L10N_CODE="pt"
-          #;;
-        esac
+        L10N_CODE="${L10N_BRANCH//dev-/}"
+        L10N_DIR="./content/${L10N_CODE}/"
```

Comment:
Hi @electrocucaracha 

I'm glad you're interested in this workflow even though it's being developed and tested :-)

According to my analysis of dev branches, code, and directory, **most localization teams use double-digit characters except in Portuguese and traditional Chinese.**

Let me explain this with examples.
- In Korean localization, `ko` is used in all parts, such as **`dev-ko`, `ko`, and `./content/ko`**.
- But, in Portuguese localization, `pt` and `pt-br` are used, such as **`dev-pt`, `pt`, and `./content/pt-br`**.

Therefore, it seems necessary to distinguish between the localization directory and code like the previous script.

Could you consider this and revise it?

Reply:
Yeah, I noticed this exception on Portuguese. The Spanish team has consolidated efforts (**`es-mx`**, **`es-ar`**, **`es-es`**, etc.) into a single language collaborative effort. We believe this approach can provide us better short and long term benefits. 

Anyway, I can modify the code to consider that exception and enable this workflow for the rest of the branches or, leave it in this way, enabling branch by branch and try to address that country exception.

Reply:
@electrocucaracha 

The Spanish team's approach looks great to me. I also like the concise code you suggested.

However, I prefer the first of the two options you suggested.
This is because the deleted code was designed to reflect the current status of all other teams.

```diff
        # Set L10n directory and code
-        case "${L10N_BRANCH}" in
-          dev-ko)          
-          L10N_DIR="./content/ko/"
-          L10N_CODE="ko"
-          ;;
-          
-          #dev-pt)
-          #L10N_DIR="./content/pt-br/"
-          #L10N_CODE="pt"
-          #;;
-        esac
+        L10N_CODE="${L10N_BRANCH//dev-/}"
+        L10N_DIR="./content/${L10N_CODE}/"
```